app/.storybook/main.js

```diff
@@ -15,12 +16,19 @@ module.exports = {
       },
     },
   ],
-  core: {
-    builder: 'webpack5'
-  },
+
   features: {
     // Emotion11 quasi compatibility issue with storybook. Disabling feature flag to support emotion11.
     // https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#emotion11-quasi-compatibility
     emotionAlias: false,
   },
+
+  framework: {
```

Comment:
Is this `core` -> `framework` change a result of upgrading storybook v6 to v7?

Reply:
Correct.